app/controllers/users_controller.rb

```diff
@@ -29,21 +29,16 @@ def create
 
   def edit
     @genres = Genre.all
-    @musician_skill = MusicianSkill.all
+    @musician_skills = MusicianSkill.all - current_user.approved_musician_skills
     @user = current_user
     @id = current_user.id
```

Comment:
why can't you call `@user.id` in view instead of creating `@id` instance variable?